docs/user-documentation/moderne-platform/getting-started/running-your-first-recipe.md

```diff
@@ -49,7 +49,7 @@ After you've logged in and granted the appropriate permissions, you'll arrive at
 
 If you went to run a recipe right now, you would find that some repositories are already selected to run against. In public Moderne, the `Default` organization is selected (a diverse selection of open-source repositories). In an enterprise instance of Moderne, the `All` organization is selected if you haven't set up an organizations service (meaning all recipes will run against every repository in the tenant). If the organizations service has been set up, it will be whatever that service returns for your user.
 
-If you'd prefer to run against a different group of repositories so that you can get relevant results more quickly, you can: select the organization that matches your needs, set up the [Organization service](../../../administrator-documentation/moderne-platform/how-to-guides/organizations-service.md) (for enterprise users), or follow our [instructions for defining user configured organizations](../how-to-guides/managing-user-configured-organizations.md#how-to-create-a-user-configured-organization).
+If you'd prefer to run against a different group of repositories so that you can get relevant results more quickly, you can: select the organization that matches your needs, set up an [Organization hierarchy](../../../administrator-documentation/moderne-platform/how-to-guides/agent-configuration/configure-organizations-hierarchy.md) (for enterprise users), or follow our [instructions for defining user configured organizations](../how-to-guides/managing-user-configured-organizations.md#how-to-create-a-user-configured-organization).
```

Comment:
This is the first time I've seen _enterprise users_. Is that the term we use for paid accounts?

Reply:
That was my understanding but if you feel like there's a better term please feel free to throw it out there.